app/static/uv.min.css

```diff
@@ -0,0 +1,19 @@
+/* stylelint-disable */
```

Comment:
Ok so we went with serving the uv css file ourselves but leaving the js to coem from the external cdn.

Remind me, what benefit do we get fro serving this file ourself

Reply:
There's no difference to serving this file ourself as it was previously set to a specific version from the CDN. This changed during testing / development to try and debug the issues with the CSP. For the sake of simplicity both files are being pulled from the cdn.